rclcpp/include/rclcpp/duration.hpp

```diff
@@ -26,11 +26,22 @@ namespace rclcpp
 class RCLCPP_PUBLIC Duration
 {
 public:
+  /// Duration constructor.
+  /**
+   * Initializes the time values for seconds and nanoseconds individually.
+   * Large values for nsecs are wrapped automatically with the remainder added to secs.
+   * Both inputs must be integers.
+   * Durations can be negative.
```

Comment:
Really, just the seconds component can be negative, nanoseconds is an unsigned int